bindings/java/src/main/native/endpoint.cc

```diff
@@ -12,8 +12,15 @@
 static void error_handler(void *arg, ucp_ep_h ep, ucs_status_t status)
 {
     JNIEnv* env = get_jni_env();
-    JNU_ThrowExceptionByStatus(env, status);
-    ucs_error("JUCX: endpoint error handler: %s", ucs_status_string(status));
+
+    ucs_debug("JUCX: endpoint %p error handler: %s", ep, ucs_status_string(status));
+    jobject jucx_ep_error_hndl = reinterpret_cast<jobject>(arg);
+    jclass jucx_ep_error_hndl_cls = env->GetObjectClass(jucx_ep_error_hndl);
+    jmethodID on_error = env->GetMethodID(jucx_ep_error_hndl_cls, "onError",
```

Comment:
is it possible to have race condition when the endpoint is destroyed from one thread, and progress calls the error handler from another thread, resulting in access to invalid object?